src/test/resources/features/complaints/ComplaintsSend.feature

```diff
@@ -50,6 +50,16 @@ Feature: Complaints Send
     And I submit the SMC Send stage
     Then the case should be closed
 
+  @ComplaintsWorkflow @ComplaintsRegression
+  Scenario: User can complete the Send stage for a BF case
+    Given I am logged into "CS" as user "BF_USER"
+    When I create a "BF" case and move it to the "Send" stage
```

Comment:
Please use 'I get a "BF" case at the "Send" stage'. Then you wont require the second step about loading and claiming the case.